Combat/Include/Monster.hpp

```diff
@@ -0,0 +1,18 @@
+#pragma once
+#include "Card.hpp"
+#include "Points.hpp"
+#include <string>
+
+
+class Monster: public Card
+{
+public:
+   Points _hp;
+   Points _strength;
+   enum class State {active, inactive};

```

Comment:
enum class trzeba wyciągnąć poza class, bo inaczej nie będzie dostępu z zewnątrz do tego. Inne pytanie to po co nam to jest? Zakładam, że to obrazuje czy karta jest na stole. Ale to po prostu stół o tym ma wiedzieć, a nie karta.